lib/engine/config/game/g_1822.rb

```diff
@@ -0,0 +1,1647 @@
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "E6",
+      "color": "crBlue"
+    },
+    {
+      "sym": "GWR",
+      "name": "Great Western Railway",
+      "logo": "1822/GWR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "M38",
+      "city": 1,
+      "color": "gwrGreen"
+    },
+    {
+      "sym": "LBSCR",
+      "name": "London, Brighton and South Coast Railway",
+      "logo": "1822/LBSCR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "M38",
+      "city": 0,
+      "color": "lbscrYellow",
+      "text_color": "black"
+    },
+    {
+      "sym": "LNWR",
+      "name": "London and North West Railway",
+      "logo": "1822/LNWR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "M38",
+      "city": 3,
+      "color": "lnwrBlack"
+    },
+    {
+      "sym": "L&YR",
+      "name": "Lancashire & Yorkshire",
+      "logo": "1822/LYR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "G22",
+      "color": "lyrPurple"
+    },
+    {
+      "sym": "MR",
+      "name": "Midland Railway",
+      "logo": "1822/MR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "J29",
+      "color": "mrRed"
+    },
+    {
+      "sym": "NBR",
+      "name": "North British Railway",
+      "logo": "1822/NBR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "H5",
+      "color": "nbrBrown"
+    },
+    {
+      "sym": "NER",
+      "name": "North Eastern Railway",
+      "logo": "1822/NER",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "L19",
+      "color": "nerGreen"
+    },
+    {
+      "sym": "SECR",
+      "name": "South Eastern & Chatham Railway",
+      "logo": "1822/SECR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "M38",
+      "city": 5,
+      "color": "secrOrange"
+    },
+    {
+      "sym": "SWR",
+      "name": "South Wales Railway",
+      "logo": "1822/SWR",
+      "tokens": [
+        0
+      ],
+      "type": "major",
+      "float_percent": 20,
+      "always_market_price": true,
+      "coordinates": "H33",
+      "color": "swrGray",
+      "text_color": "black"
+    }
+  ],
+  "trains": [
+    {
+      "name": "L",
```

Comment:
I am not sure if the code supports the L train.

Reply:
No i just wanted as much as i can in the config. I will start to handle the accual code now.

Reply:
Probably not, but i'd suggest a separate PR for this

Reply:
I figured it doesnt matter if it doesnt work at the moment as nothing else works as it should. But i can remove the L train along with the phase 1.